app/index.html

```diff
@@ -15,10 +15,10 @@
 </head>
 <body>
     <nav class="navbar">
-        <a href="index.html" class="logo">
+        <a ui-sref="landing" class="logo">
```

Comment:
An `href` wouldn't matter here because internally they are capturing the `click` event and handling it, effectively cancelling out the transition to a new url